packages/exchange/src/pods/demand/create-demand.dto.ts

```diff
@@ -37,4 +37,7 @@ export class CreateDemandDTO {
 
     @IsBoolean()
     public readonly boundToGenerationTime: boolean;
+
+    @IsBoolean()
```

Comment:
Please mark this field as Optional() as well, this would allow us to retain the compatibility if previous API deployments. 

Also, the default version used by the system should be `true` as it was previously.